Branch-TestBed/Branch-TestBed/AppDelegate.m

```diff
@@ -46,7 +46,12 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // Required. Initialize session. automaticallyDisplayDeepLinkController is optional (default is NO).
     [branch initSessionWithLaunchOptions:launchOptions automaticallyDisplayDeepLinkController:YES deepLinkHandler:^(NSDictionary *params, NSError *error) {
         if (!error) {
+
             NSLog(@"initSession succeeded with params: %@", params);
+            NSDictionary *searchAds = [BNCPreferenceHelper preferenceHelper].appleSearchAdDetails;
+            NSLog(@"Search Ads:\n%@.", searchAds);    //  Apple Search ad demo.
+            NSLog(@"");
```

Comment:
Just to make sure we are on the same page, I am talking about the empty string one. `NSLog(@"");`